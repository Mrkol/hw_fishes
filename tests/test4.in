1
##########
#........#
#.FFFCCC.#
#.FFFCCC.#
#.FFFCCC.#
#........#
##########
