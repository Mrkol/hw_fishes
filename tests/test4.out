##########
#..F..C..#
#.F.FC.C.#
#F......C#
#.F.FC.C.#
#..F..C..#
##########
